modules/books.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-restricted-globals */
```

Comment:
- [ ] It will be great to fix the linters errors instead of disabling them. Remember, linters helps you write clean code.